### ISSUE TITLE
string variable containing a single backslash

### ISSUE THREAD

**Issue by bruceravel** [xraypy/xraylarch#95]
There seems to be no way in larch to make a string that is one character long with that character being a backslash.  All of the following work in the python repl and none work in larch:

```python
>>> a = '\\'
>>> print a
\
>>> a = "\\"
>>> print a
\
>>> a = u'\\'
>>> print a
\
```
In larch, each one returns the `....>` prompt and eventually complains with `SyntaxError: EOL while scanning string literal` when enough quotes are entered.

The best thing I have thought of is to make a string like so:
```python
a = "\\ "
b = a[0:1]
```
which is fine, but awkward and surprising.

**Comment by newville** [comment]
fixed in 8c720377594e9a8c0fba3fe9493fba07a0e08988.   The joys of string handling!

**Comment by bruceravel** [comment]
Fixed indeed.  Cool!
